aidants_connect_habilitation/forms.py

```diff
@@ -104,6 +104,25 @@ class OrganisationRequestForm(PatchedErrorListForm):
         required=False,
     )
 
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+        self.__widget_attrs(
+            "type_other",
+            {
+                "data-dynamic-form-target": "typeOtherInput",
+                "data-displayed-label": "Veuillez précisez le type de structure",
```

Comment:
```suggestion
                "data-displayed-label": "Veuillez préciser le type de structure",
```

Un jour, quelqu'un m'a dit « Voulez-vous prenez un thé ? » et depuis je ne fais plus la faute. :'D

Reply:
Oui, bon, ben j'ai écrit un peu vite sans me relire quoi :eyes:

Reply:
Est-ce que ce serait pas même :

```suggestion
                "data-displayed-label": "Veuillez préciser le type d'organisation",
```